crispy_forms/base.py

```diff
@@ -1,3 +1,11 @@
+from __future__ import annotations
+
+from types import TracebackType
+from typing import Collection, Union
```

Comment:
If you wrap these inside `if typing.TYPE_CHECKING:` they will only be imported on `mypy` runs.